frontend/server/src/main/java/org/pytorch/serve/http/InferenceRequestHandler.java

```diff
@@ -179,7 +179,12 @@ private void predict(
         Job job = new Job(ctx, modelName, modelVersion, WorkerCommands.PREDICT, input);
         if (!ModelManager.getInstance().addJob(job)) {
             throw new ServiceUnavailableException(
-                    "No worker is available to serve request: " + modelName);
+                    "Model \""
+                            + modelName
+                            + "\" Version "
+                            + modelVersion
```

Comment:
Add a null check for modelVersion and return default version in that case.

Reply:
Now instead of null, it will show defaultVersion